.github/workflows/ci.yml

```diff
@@ -87,3 +65,44 @@ jobs:
         uses: shogo82148/actions-goveralls@v1
         with:
           path-to-profile: coverage-all.out
+
+  testdata:
+    name: Verify testdata directory
+    needs: lint
+    runs-on: ubuntu-latest
+    # Pull requests from the same repository won't trigger this checks as they were already triggered by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.15'
+      # This step is needed as the following one tries to remove
+      # kustomize for each test but has no permission to do so
+      - name: Remove pre-installed kustomize
+        run: sudo rm -f /usr/local/bin/kustomize
+      - name: Verify testdata directory
+        run: make check-testdata
+
+  debug:
+    name: Debug failing workflows
+    needs: # Add to this list every job in this workflow
+      - lint
+      - test
+      - coverage
+      - testdata
+    runs-on: ubuntu-latest
+    # Failed PRs are shown in their check but pushes are hard to notice so we create an issue
+    if: failure() && github.event_name == 'push'
```

Comment:
This should probably only run on master.

Reply:
Reason?
That would prevent it running on `release-X` branches for example, and we can always close the issue easily

Reply:
I'm thinking of all the branches existing in this repo having issues created each time the branch is broken and someone pushes, which is possibly often on non-trunk dev branches. Will the debug action recreate an issue for each push, or only once for a broken branch?

Reply:
It will recreate one per push, yes.
In relation to all those branches, I created several issues recently tracking some changes that should be done to clean up a bit the tree, like removing some branches, or moving some to `kubebuilder-release-tools` as we commented in a meeting. I don't have admin privileges to delete branches though.

Reply:
Done.